rules/cabal-vehicles.yaml

```diff
@@ -393,78 +393,47 @@ TARANTULA:
 SILVER:
 	Inherits@1: ^VoxelVehicle
 	Inherits@2: ^HoverVehicle
-	Inherits@FeatureTargettype: ^FeatureTargettypeAircraft
```

Comment:
No TargetType means that the unit can't be attacked. Is this really what you want?